widgets/nuxeo-selectivity.html

```diff
@@ -613,6 +640,38 @@
           });
         }
 
+        _getScrollParent() {
+          let scrollParent = document.body; // use the document.body as the default scrollable parent
+          let style = getComputedStyle(this);
+          const excludeStaticParent = style.position === 'absolute';
+          const overflowRegex = /(auto|scroll)/;
+
+          if (style.position !== 'fixed') {
+            let parent = this;
+            while (parent) {
+              if (parent.parentElement) {
+                parent = parent.parentElement;
+              } else if (parent.getRootNode()) {
+                parent = parent.getRootNode();
+                if (parent) {
```

Comment:
[Same thing as in the main PR.](https://github.com/nuxeo/nuxeo-elements/pull/312#discussion_r525234544)